graphql/getMoreReviewedApplications.graphql

```diff
@@ -0,0 +1,56 @@
+query getMoreReviewedApplications(
+    $grantId: String!
+    $reviewerId: Bytes!
+    $first: Int
+    $skip: Int
+) {
+    grantApplications(
+        where: {
+            grant: $grantId,
+            doneReviewerAddresses_contains_nocase: [$reviewerId]
+        },
+        first: $first,
+        skip: $skip,
+    ) {
+        id
+        state
+        createdAtS
+        applicantId
+        milestones {
+            amount
+        }
+        reviews {
```

Comment:
Can filter by reviewerAddress here as well since you only need the signed in user's review?